skypy/galaxies/vdf_sampler.py

```diff
@@ -0,0 +1,92 @@
+"""Sample from velocity distribution function
+"""
+
+import numpy as np
+import scipy.special as special
+from scipy.interpolate import interp1d
+
+
+def sample_vdf(x_min, x_max, resolution=100, size=1):
+    """Sample from velocity dispersion function of elliptical galaxies in the local universe [1]_.
```

Comment:
Please add the equation in LaTeX style to the docstring in a `Note` section and refer to the equation number of the paper. This makes it easier to understand which equation is implemented. See for example docstring of [skypy.galaxies.redshift.py.smail](https://github.com/skypyproject/skypy/blob/df2536974c122dd72498f8d146913c14c832833b/skypy/galaxies/redshift.py#L27).

Reply:
Done.